tmp-frontend/Gemfile

```
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gemspec
+
+group :development, :test do
+  gem 'rake', '~> 13.0.1'
+  gem 'pry-byebug'
```

RuboCop (1.76.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pry-byebug` should appear before `rake`.

(Bundler/OrderedGems)